index.html

```diff
@@ -2412,7 +2412,17 @@ <h2>Versioning</h2>
 			<p class="issue" data-number="12">See corresponding open issue.</p>
 
 	</section>
-	
+
+	<section id="vdr-forks">
+		<h2>VDR Network Forks</h2>
+		<p>DID Methods that utilize VDRs of distributed systems (such as distributed ledgers or other distributed systems) need to manage the potential for network forks taking place. A <a>DID method</a> that uses a distributed system as a <a>verifiable data registry</a> SHOULD disambiguate the VDR they are using in the case of such forks by either:</p>
+		<ul>
+			<li>Referencing a well-known public source or resolution library that can distinguish between forked networks in the did method specification</li>
+			<li>Maintaining a manual mapping in the did method specification of the method-specific-id subdivisions to the various VDRs</li>
+			<li>Adding a new did parameter to specify which VDR is being used when network forks occur</li>
+		</ul>
+		<p>In addition a <a>DID Resolver</a> that is aware of any VDR network forks SHOULD warn users that these forks may affect resolution results.</p>
```

Comment:
```suggestion
		<p>In addition a <a>DID resolver</a> that is aware of any VDR network forks SHOULD warn users that these forks may affect resolution results.</p>
```

Reply:
I think I agree with this, but does that mean this spec should be defining such a warning?